source/guides/network/dpdk.rst

```diff
@@ -32,13 +32,13 @@ This example uses the following DPDK components:
 Prerequisites
 *************
 
-*  Two platforms using |CL-ATTR| release `13330`_ or higher.
+*  Two platforms using |CL-ATTR| release `31130`_ or higher.
 *  Both images must include the :command:`kernel-native` bundle.
 *  Install the :command:`network-basic-dev` bundle with the command:
 
    .. code-block:: bash
 
-      sudo swupd bundle-add network-basic-dev
+      sudo swupd bundle-add dpdk devpkgdpdk
```

Comment:
I did not test without `network-basic-dev` and I was not very clear in my feedback. Let's make this `sudo swupd bundle-add network-basic-dev dpdk devpkg-dpdk` (note typo in your fix)